great-idea-website/css/index.css

```diff
@@ -63,4 +63,122 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+hr {
+	width: 80%;
+}
+.main-content {
+	margin: 0 20%;
+}
+
+nav {
+	width: 70%;
+	display: inline-block;
+	margin: 0 0 0 5%;
+
+}
+nav a {
+	text-decoration: none;
+	display: inline-block;
+	margin: 5%;
+}
+
+.logo {
+	display: inline-block;
+}
```

Comment:
To achieve better centering of the logo with the nav links in this exercise, you can manipulate the margins to push the image around. Of course moving forward, you are much better off using Flexbox to align and center things.